master/buildbot/data/changes.py

```diff
@@ -42,6 +42,9 @@ def _fixChange(self, change, is_graphql):
                     {'name': k, 'source': v[1], 'value': json.dumps(v[0])}
                     for k, v in props.items()
                 ]
+                change['builds'] = yield self.master.db.builds.getBuildsForChange(
```

Comment:
I think this will be slow when retrieving many changes. Let's skip it for now, especially if you aren't using graphql.

Reply:
Ok, I see. Probably it does't make sense to always fetch them. But can it be at least added when `branch` is specified? The reason the builds are needed is to directly show them in the grid view without any additional calls. Maybe they can be merged in a single query. What do you think would be the best approach? Also, if there are two many changes, we can leave it out for now and I'll work on a new PR to add this.